util/xs_scripts/kmh_v3_btb.sh

```
@@ -7,4 +7,6 @@ for var in GCBV_REF_SO GCB_RESTORER gem5_home; do
     checkForVariable $var
 done
 
-$gem5 $gem5_home/configs/example/kmhv3.py --generic-rv-cpt=$1
\ No newline at end of file
+$gem5 $gem5_home/configs/example/kmhv3.py --generic-rv-cpt=$1 \
```

Shellcheck (0.11.0)

[warning] gem5 is referenced but not assigned.

(SC2154)